integration-tests/pipelines/e2e-main-pipeline.yaml

```diff
@@ -117,18 +113,30 @@ spec:
 
               pids=()
 
-              while IFS= read -r version; do
+              while IFS= read -r config; do
+                OCP_VERSION=$(echo "$config" | jq -r '.[] | select(.key == "OCP").value')
+                ACS=$(echo "$config" | jq -r '.[] | select(.key == "ACS").value')
+                REGISTRY=$(echo "$config" | jq -r '.[] | select(.key == "Registry").value')
+                TPA=$(echo "$config" | jq -r '.[] | select(.key == "TPA").value')
+                SCM=$(echo "$config" | jq -r '.[] | select(.key == "SCM").value')
+                PIPELINE=$(echo "$config" | jq -r '.[] | select(.key == "Pipeline").value')
+
                 (
                 pipeline_run_name=$(tkn pipeline start -f https://raw.githubusercontent.com/$REPO_ORG/rhtap-cli/refs/heads/$BRANCH/integration-tests/pipelines/rhtap-cli-e2e.yaml \
-                  --param ocp-version="$version"\
+                  --param ocp-version="$OCP_VERSION"\
                   --param job-spec="$JOB_SPEC"\
                   --param konflux-test-infra-secret="$(params.konflux-test-infra-secret)" \
+                  --param acs_config="$ACS" \
+                  --param registry_config="$REGISTRY" \
+                  --param tpa_config="$TPA" \
+                  --param scm_config="$SCM" \
+                  --param pipeline_config="$PIPELINE" \
                   --use-param-defaults \
                   --labels "appstudio.openshift.io/component=${KONFLUX_COMPONENT_NAME}" \
                   --labels "appstudio.openshift.io/application=${KONFLUX_APPLICATION_NAME}" \
                   --labels "pipelines.appstudio.openshift.io/type=$(context.pipelineRun.name)" \
                   --labels "test.appstudio.openshift.io/scenario=pr-e2e-tests" \
-                  --prefix-name "e2e-$version"\
+                  --prefix-name "e2e-$OCP_VERSION"\
```

Comment:
If there is multiple configuration gets generated with just say two configuration for 4.16, How this plr will be differentiated?

Reply:
ah, good catch

Reply:
This is  prefix-name, the pipeline name will append a random string, for example `e2e-4.16-rkx6m`.